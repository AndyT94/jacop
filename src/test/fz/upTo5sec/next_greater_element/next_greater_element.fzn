var 7..7: var1:: output_var;
var 8..8: var2:: output_var;
array [1..4] of var int: variables:: output_array([1..4]) = [3,5,8,9];
solve  satisfy;
