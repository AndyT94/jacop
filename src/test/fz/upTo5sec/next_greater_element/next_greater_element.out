var1 = 7;
var2 = 8;
variables = array1d(1..4, [3, 5, 8, 9]);
----------